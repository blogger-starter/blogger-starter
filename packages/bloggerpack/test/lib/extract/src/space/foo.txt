<extract-start>
  <foo>foo</foo>

          <foo>foo</foo>
</extract-end>
