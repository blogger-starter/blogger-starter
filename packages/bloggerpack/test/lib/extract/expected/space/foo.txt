/* src/space/foo.txt */

<foo>foo</foo>

          <foo>foo</foo>
